cedar-dafny/validation/validator.dfy

```diff
@@ -42,12 +42,12 @@ module validation.validator {
       set a,p,r | a in actionIds.Keys &&
                   p in actionIds[a].appliesTo.principalApplySpec &&
                   r in actionIds[a].appliesTo.resourceApplySpec ::
-        RequestType(p, a, r, actionIds[a].context)
+        RequestType(p, a, r, RecordType(actionIds[a].context, false))
```

Comment:
Are we assuming that actions contexts in the schema are complete? I feel like it's more natural to treat the context in the schema as what is *required* for an action, which would mean that we should treat the context as open.